### DISCUSSION TITLE
Vent via wireless etc

### DISCUSSION THREAD

**Discussion by MrPeens** [gpstar81/GPStar-proton-pack#273] [Q&A] [answered]
Just updated to v4.4 firmware on pack. Don't have my wand built yet so pack is operating on its own. 

1)Seems since the update my pack will turn on as soon as the battery is turned on regardless of the ion arm switch position. It didn't do this before. Previously it would only turn on when the ion arm switch was flipped (after battery was on obviously)

2)When using wireless, the vent option is greyed out. I'd like to test the smoke machine with the vent button via wireless. When I try to access the smoke options via wireless it gives me an error saying it couldn't retrieve settings. Are the smoke settings stored in the wand board? Do I need the wand connected in order to trigger venting via wireless??

**Comment by gpstar81** [comment, marked as answer]
#1.  There is a eeprom setting called demo light mode which does this feature. It is possible you have it enabled.

#2. Yes the wand triggers the overheat sequence.
